<ui version="4.0" >
 <class>ExpDescriptionEditor</class>
 <widget class="QWidget" name="ExpDescriptionEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Experiment Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="mntGrpTab" >
      <attribute name="title" >
       <string>Measurement Group</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Active Measurement Group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="activeMntGrpCB" >
           <property name="toolTip" >
            <string>Selects the active Measurement Group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="createMntGrpBT" >
           <property name="toolTip" >
            <string>Create a new Measurement Group</string>
           </property>
           <property name="text" >
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="deleteMntGrpBT" >
           <property name="toolTip" >
            <string>Delete the current Measurement Group</string>
           </property>
           <property name="text" >
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="plotsButton" >
           <property name="toolTip" >
            <string>If checked, the configured plots are shown</string>
           </property>
           <property name="text" >
            <string>plots</string>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="MntGrpChannelEditor" name="channelEditor" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Snapshot Group</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6" >
       <item>
        <widget class="QSplitter" name="splitter" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTabWidget" name="tabWidget_2" >
          <property name="currentIndex" >
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_4" >
           <attribute name="title" >
            <string>Sardana Elements</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5" >
            <item>
             <widget class="SardanaElementTreeWidget" name="sardanaElementTree" />
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab" >
           <attribute name="title" >
            <string>External (Tango)</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4" >
            <item>
             <widget class="TaurusModelSelectorTree" name="taurusModelTree" />
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="layoutWidget" >
          <layout class="QVBoxLayout" name="verticalLayout_2" >
           <item>
            <widget class="QLabel" name="label_5" >
             <property name="text" >
              <string>(Drag elements from the above selectors and drop them at the bottom list)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="TaurusModelList" name="preScanList" />
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Storage</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout" >
       <property name="fieldGrowthPolicy" >
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="toolTip" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">comma separated list of file names in which to store the results of the scans. Use .h5 extension for NeXus files (preferred) and .dat for spec format (note: SPEC format is &lt;span style=" font-weight:600;">not &lt;/span>supported)&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="text" >
          <string>File Name(s)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="filenameLE" >
         <property name="toolTip" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">comma separated list of file names in which to store the results of the scans. Use .h5 extension for NeXus files (preferred) and .dat for spec format (note: SPEC format is &lt;span style=" font-weight:600;">not &lt;/span>supported)&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Path</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QWidget" native="1" name="widget" >
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="pathLE" />
          </item>
          <item>
           <widget class="QToolButton" name="choosePathBT" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Data compression</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <widget class="QComboBox" name="compressionCB" >
         <item>
          <property name="text" >
           <string>nowhere</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>for all datasets</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>for datasets of rank 1 or more</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>for datasets of rank 2 or more</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>for datasets of rank 3 or more</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaurusModelList</class>
   <extends>QListView</extends>
   <header>taurus.qt.qtgui.panel</header>
  </customwidget>
  <customwidget>
   <class>TaurusModelSelectorTree</class>
   <extends>TaurusWidget</extends>
   <header>taurus.qt.qtgui.panel</header>
  </customwidget>
  <customwidget>
   <class>SardanaElementTreeWidget</class>
   <extends>QMainWindow</extends>
   <header>sardana.taurus.qt.qtgui.extra_sardana</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MntGrpChannelEditor</class>
   <extends>QMainWindow</extends>
   <header>sardana.taurus.qt.qtgui.extra_sardana</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
